### DISCUSSION TITLE
Is there a way to uninstall?

### DISCUSSION THREAD

**Discussion by kima111** [harlan-zw/unlighthouse#59] [Q&A] [answered]
Looking for a way to uninstall Unlighthouse. I can't seem to find what it installed and where it installed.

**Comment by ManAnRuck** [comment, marked as answer]
You can use your terminal to check where it is installed:
Unix: `which unlighthouse`
Windows: `where unlighthouse` (not sure if where is correct)

for me it's currently `/Users/MyName/.nvm/versions/node/v31.1.3/bin/unlighthouse`

Then you should see how you installed it and can use the shown package manager in the path to uninstall.

**Reply by harlan-zw** [reply]
Good answer, thanks!

**Comment by harlan-zw** [comment]
Just to clarify, the installation is based on how _you_ installed it. If you're using the default, which is `npx`, then you should read up on how `npx` works: https://docs.npmjs.com/cli/v9/commands/npx/

Let me know if that isn't clear
